### PR TITLE
Feat added cart page functionality

### PR DESCRIPTION
**Pull request by raman1999**

### CODE REVIEW

File: src/Utils/updateCart.jsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        var { data } = await axios.delete(
+          `api/user/cart/${cartItem._id}`,
+          postHeader
+        );
+
+        dispatch({
+          type: "UPDATE_CART",
+          payload: { cart: data.cart, toastMsg: "Item Moved Successfully" },
+        });
+        break;
```
Comment (rahulcs754) [inline src/Utils/updateCart.jsx:55]: why do we need a break? It is necessary to have

**Review by rahulcs754 — APPROVED** [review]
code looks good. 
I think if we are not using break then it is no issue.

File: src/Utils/updateCart.jsx (hunk 1 above)
Comment (raman1999) [reply src/Utils/updateCart.jsx:55]: yes , it's needed , as switch search for first match case , and if no break is provided in that case , it also go to the next case and continues till break found

**Review by aishwarya-bm — APPROVED** [review]
code looks clean,readable, and concise. Good to merge

**Pull request merged**